cmd/_dbxroot/cmd/install-to-disk.go

```diff
@@ -15,13 +16,16 @@ var installToDiskCmd = &cobra.Command{
 	Use:   "install-to-disk",
 	Short: "Install Dogebox to a disk.",
 	Long: `Install Dogebox to a disk.
-This command requires --disk and --dbx-secret flags.
-
 Example:
-  _dbxroot install-to-disk --disk /dev/sdb --dbx-secret ?`,
+  _dbxroot install-to-disk [--variant generic] --disk /dev/sdb --dbx-secret ?
```

Comment:
`[--variant generic]` isn't really optional - or at least it defaults to `""`, we should align this or make it actually-required.